kornia/models/edge_detector/dexined.py

```diff
@@ -26,10 +26,8 @@ def to_onnx(
         image_size: Optional[int] = 352,
     ) -> Tuple[str, EdgeDetector]:
         edge_detector = DexiNedBuilder.build(pretrained, image_size)
-        model_name = "DexiNed"
         if onnx_name is None:
-            _model_name = model_name
-            onnx_name = f"Kornia-RTDETR-{_model_name}-{image_size}.onnx"
+            onnx_name = f"Kornia-DexiNed-{image_size}.onnx"
```

Comment:
```suggestion
            onnx_name = f"kornia_dexined_{image_size}.onnx"
```
can we default using lowercase as it is case sensitive plus underscores ?